README.md

```diff
@@ -0,0 +1,190 @@
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+
+
+**[Portfolio]** is a Microverse Project bootcamp to learn about Mobile first and responsive desing.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- JavaScript
+- GitHub
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Mobile Designer]**
+- **[Headline]**
+- **[Tourism page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Click Here](https://ulises2607.github.io/cayafate-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- Any internet browser.
+- Git or Github for desktop.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git@github.com:ulises2607/cayafate-project.git
+```
+
+
+### Install
+
+Install this project with:
+
+- N/A
+
+### Usage
+
+-N/A
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx hint .
+```
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+```sh
+  npx eslint .  
+```
+
+### Deployment
+
+#### If you want to deploy the website you can use the free webportals or platforms for deloyment. Like `gh-pages`, `vercel`, `netlify`and so many other free platforms. This website is deploye using `gh-pages`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@ulises2607](https://github.com/ulises2607)
+- LinkedIn: [Cesar Ulises Cappa Subelza](https://www.linkedin.com/in/cesar-ulises-cappa-subelza/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [1] **[Popups]**
+- [2] **[Animations]**
+- [3] **[Images cards]**
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+- Microverse Code Reviewers
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please comment on what you liked about it, and what I can improve on to make it better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank microverse and my code reviewers team for provide me the tools to make this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please 🙏🏻  add the `MIT` license properly because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .

[Screencast from 08-07-23 04:07:27 PM IST.webm](https://github.com/ulises2607/cayafate-project/assets/66476329/2d84ce07-c290-43a7-9c11-33bc43652341)